package.json

```diff
@@ -41,6 +41,7 @@
   "dependencies": {
     "sequelize": "^3.30.2",
     "sequelize-cli": "^2.5.1",
+    "split.js": "1.3.5",
```

Comment:
@johnmhoran It looks like the grip can be included with base64. With that said, I'm not a big fan of the grip. I see the cursor changes to a `ew-resize` cursor when hovering over the splitter to indicate it can be dragged.